docs/integration/infra.md

```diff
@@ -0,0 +1,276 @@
+
+
+## How it works
+
+A Golang-based [Sematext Agent](../agents/sematext-agent) will be installed on the host you want to monitor. This agent sends all infrastructure data to an Infra App whose token is configured during the Agent's installation.
+
+It is capable of running in two kinds of environments:
+- bare-metal, virtual machine - we provide rpm/deb packages for most popular linux distributions
+- containers - a special sematext/agent container image provides support for monitoring in container environments
+
+In both cases, installing the agent will start gathering all infrastructure data right away. Additionally, it will enable [Service Autodiscovery](../monitoring/autodiscovery) and [Logs Autodiscovery](../logs/discovery/intro/) features which make it possible to automatically start monitoring any of supported [Sematext integrations](./index) or to ship logs produced by any process or container.
+
+### eBPF Support
+
+To gain deep **insight into the Linux kernel**, Sematext Agent relies on **eBPF** to implant **instrumentation points**, which means to **attach eBPF programs to kprobes** on kernel functions. This ensures a very efficient and powerful system exploration approach with better network tracing and negligible overhead.
+
+
+## Enabling Infra Monitoring
+
+If you've set up Sematext Monitoring at some point in the past, you can easily enable Infra Monitoring by:
+
+- [Upgrading](./spm-faq/#agent-updating) your `sematext-agent` to the latest version
+- Setting up Infra App token on each host with Sematext Agent by running the `setup-infra` command (in bare-metal/virtual machine environments)
+
+> **Note**: For container monitoring, we suggest uninstalling the existing deprecated Agent and going through the new setup steps for Docker Monitoring in Sematext Cloud.
+
+When doing a clean installation, regardless of whether you're installing in containerized environment or not, follow monitor installation instructions displayed in Sematext Cloud. Those instructions will always include everything needed to set up an Infra App, either on its own or alongside some other App Agent.
+
+
+### The `setup-infra` command
+
+This command is applicable only in bare-metal/virtual machine environments.
+
+The `setup-infra` command is used to set the Infra App to which Sematext Agent should send data it collects.
+
+To see which Infra Apps exist in your account visit Sematext Cloud -> Apps. You can choose any of the existing Infra Apps and go to App Settings > Tokens or you can create a new Infra App.
+
+Once you do that you'll have the token of that particular App. To configure the Sematext Agent to start sending data to a particular Infra App run `setup-infra` on each host you wish to update:
+
+`sudo bash /opt/spm/bin/setup-infra --infra-token YOUR_INFRA_APP_TOKEN_HERE`
+
+
+### Enabling Infra Monitoring in Docker environment
```

Comment:
```suggestion
### Enabling Infra Monitoring in Container environments
```

Maybe better to say `Container` instead, and add a section for K8s.

Reply:
We have k8s instructions elsewhere, so wouldn't want to repeat all that. But Docker/Swarm also exist in other places. So, either remove all from here or add a duplicate of k8s instructions?